frontend/spec/controllers/spree/checkout_controller_spec.rb

```diff
@@ -54,6 +54,7 @@
         # The first step for checkout controller is address
         # Transitioning into this state first is required
         order.update_column(:state, "address")
+        stub_spree_preferences(associate_user_in_authentication_extension: false)
```

Comment:
How is it that we need to override the new setting value here? Wasn't the method useless?